lxd/network/acl/driver_common.go

```diff
@@ -752,32 +752,45 @@ func (d *common) Delete() error {
 func (d *common) GetLog(clientType request.ClientType) (string, error) {
 	// ACLs aren't specific to a particular network type but the log only works with OVN.
 	logPath := shared.HostPath("/var/log/ovn/ovn-controller.log")
+	logEntries := []string{}
 	if !shared.PathExists(logPath) {
-		return "", fmt.Errorf("Only OVN log entries may be retrieved at this time")
-	}
-
-	// Open the log file.
-	logFile, err := os.Open(logPath)
-	if err != nil {
-		return "", fmt.Errorf("Couldn't open OVN log file: %w", err)
-	}
+		// Check if this is OVN builtin of if OVN is provided by a MicroOVN deployment.
+		targetPath, err := os.Readlink("/run/openvswitch")
+		if err != nil {
+			return "", fmt.Errorf("Failed to read symlink while checking for OVN logs: %v\n", err)
+		}
 
-	defer func() { _ = logFile.Close() }()
+		if !openvswitch.MicroovnTargetChassisPath.MatchString(targetPath) {
```

Comment:
Could we instead use `strings.HasSuffix(targetPath, "/microovn/chassis/switch")` instead of a `regexp` match?

LXD's snap has this in `daemon.start`:
```
elif [ -d "${SNAP_DATA}/microovn/chassis/switch" ]; then
    ln -s "${SNAP_DATA}/microovn/chassis/switch" /run/openvswitch
```

Reply:
yes correct! fixed.